src/Correlate.Core/Extensions/LoggerExtensions.cs

```diff
@@ -49,5 +49,14 @@ public KeyValuePair<string, object> this[int index]
                 throw new ArgumentOutOfRangeException(nameof(index));
             }
         }
+
+        /// <summary>
+        /// Returns a representation of the scope items as a list of comma-separated items,
+        /// matching the default representation provided by the standard Console logger.
+        /// </summary>
+        public override string ToString()
+        {
+            return string.Join(", ", this.Select(kv => $"{kv.Key}:{kv.Value}"));
```

Comment:
Since the `IReadOnlyDictionary` implementation is hardcoded to a single key/value pair, there is no need to use LINQ plus pay the iterator allocation penalty. It is this way on purpose, since the scope is created and used in a hot path.

I also believe using string concatenation in this particular case might be desirable over string interpolation. Your proposal seems very small and trivial, but if you log a lot, it really adds up. I may add a benchmark for this specific case later to be sure.

```suggestion
            return _scopeKey + ':' + _correlationId;
```